tests/test_bqm.py

```diff
@@ -329,6 +329,17 @@ def test_change_vartype_spin_to_binary_copy(self, name, BQM):
         self.assertConsistentEnergies(spin=bqm, binary=new)
 
 
+class TestClear(unittest.TestCase):
```

Comment:
Both tests look good, as far as I can tell. I guess `BQMs` goes through all valid types of BQMs, and there's no need to add tests for the base quadratic model class (should be covered by BQM and QM).

Reply:
Yup!